client/verta/verta/_deployment/endpoint.py

```diff
@@ -76,9 +77,67 @@ def _get_json_by_path(cls, conn, workspace, path):
         # TODO: iterate through response.json().get('endpoints', []) to find matching creator_request.path
 
     def update(self, run, strategy):
-        raise NotImplementedError
-        # TODO: check if isinstance(run, experimentrun.ExperimentRun)
-        # TODO: check if isinstance(strategy, deployment._UpdateStrategy)
+        if not isinstance(run, experimentrun.ExperimentRun):
+            raise TypeError("run must be an ExperimentRun")
+
+        if not issubclass(type(strategy), _UpdateStrategy):
```

Comment:
`isinstance()` is a little more direct than `issubclass()`
```suggestion
        if not isinstance(strategy, _UpdateStrategy):
```

Reply:
Fixed.